prime-router/metadata/fhir_mapping/hl7/datatype/Identifier_XON.yml

```diff
@@ -0,0 +1,49 @@
+value:
```

Comment:
Created this file to handle the portions of the XON->Organization mapping which are supposed to be nested within organization.identifier, based on [this mapping](https://docs.google.com/spreadsheets/d/1-cdMWepRbnqJOVVZ-y70Qv9Ub7evClmbX9gYO0kHmpo/edit#gid=0).